app/routes/transfer/direct/$chainId.$assetId/$address.confirmation.tsx

```diff
@@ -120,7 +129,7 @@ const Page = () => {
               <BodyText align="left" className="text-text-hint">
                 {title}
               </BodyText>
-              <MediumTitle>{value}</MediumTitle>
+              {typeof value === 'string' ? <MediumTitle>{value}</MediumTitle> : value}
```

Comment:
For me this behavior is unobvious, IMO you can fix it in 2 ways:
```tsx
<div className="text-text-medium-title">{value}</div>
```
or add `as="div"` into Typography components
```tsx
<MediumTitle as="div">{value}</MediumTitle>
```